src/components/control-bar/VolumeMenuButton.js

```diff
@@ -4,13 +4,15 @@ import classNames from 'classnames';
 
 import PopupButton from '../popup/PopupButton';
 import VolumeBar from '../volume-control/VolumeBar';
+import { mergeAndSortChildren } from '../../utils';
 
 const propTypes = {
   player: PropTypes.object,
   actions: PropTypes.object,
   vertical: PropTypes.bool,
   className: PropTypes.string,
-  alwaysShowVolume: PropTypes.bool
+  alwaysShowVolume: PropTypes.bool,
+  children: PropTypes.any
```

Comment:
What do you think about `PropTypes.node` here? Might be helpfully more specific, namely it would exclude things like functions, undefined, and objects.